shared/chat/inbox/row/small-team/top-line.js

```diff
@@ -32,14 +32,9 @@ class _SimpleTopLine extends React.Component<Props> {
   }
 
   render() {
-    const boldOverride = this.props.showBold ? Styles.globalStyles.fontBold : null
+    const boldStyle = this.props.showBold ? styles.bold : null
```

Comment:
some small changes to use styleSheets and fastBlank to fix overdraw issues in the inbox